portfolio/src/main/webapp/style.css

```diff
@@ -10,6 +10,13 @@ body{
   font-family: 'Roboto', sans-serif;
 }
 
+# content-photography {
+    margin-left: auto;
```

Comment:
There's a shorthand for margin if you want to use it in the future.

You can apply to top/bottom and left/right
`margin: 0 auto;` will apply 0 to top and bottom and auto to left and righjt

Or you can just specify all 4. Starts with top and goes clockwise:
`margin: 0 auto 0 auto;` would apply 0 to top, auto to right, 0 to bottom, and auto to left.